lib/Bio/KBase/ObjectAPI/KBasePhenotypes/PhenotypeSimulationSet.pm

```diff
@@ -73,10 +73,11 @@ sub export {
 sub printTSV {
     my $self = shift;
 	my $args = Bio::KBase::ObjectAPI::utilities::args([], {file => 0,path => undef}, @_);
-	my $output = ["geneko\tmediaws\tmedia\taddtlCpd\taddtlCpdBounds\tcustomReactionBounds\tgrowth\tsimulated growth\tsimulated growth fraction\tgapfilled reaction count\tgapfilled reactions"];
+	my  $output = ["Growth condition\tGenekKO\tmediaws\tAddtional Compounds\taddtlCpdBounds\tcustomReactionBounds\tObserved Normal growth\tSimulated growth\tsimulated growth fraction\tgapfilled reaction count\tgapfilled reactions\tPrediction class"];
```

Comment:
We should make this consistently CamelCase or Title Case

Reply:
Pretty sure there's no reimport of the data so I'd lean toward the latter

Reply:
My objective was to match the data viewer for all fields that they have in common.

Reply:
I can change them all to Title case if you want.